core/modules/qproc/QuerySession.cc

```diff
@@ -402,28 +394,32 @@ std::vector<std::string> QuerySession::_buildChunkQueries(ChunkSpec const& s) co
             std::stringstream ss;
             std::copy(sVector.begin(), sVector.end(),
                       std::ostream_iterator<ChunkSpecSingle>(ss, ","));
-            LOGF_INFO("subchunks: %1%" % ss.str());
+            LOGF(getLogger(), LOG_LVL_INFO, "subchunks: %1%" % ss.str());
         }
         typedef ChunkSpecSingle::Vector::const_iterator ChunkIter;
         for(ChunkIter i=sVector.begin(), e=sVector.end(); i != e; ++i) {
             for(QueryTplVectorIter j=queryTemplates.begin(), je=queryTemplates.end(); j != je; ++j) {
-                LOGF_DEBUG("adding query %1%" % _context->queryMapping->apply(*i, *j));
+                LOGF(getLogger(), LOG_LVL_DEBUG, "adding query %1%" % _context->queryMapping->apply(*i, *j));
                 q.push_back(_context->queryMapping->apply(*i, *j));
             }
         }
     }
-    LOGF_DEBUG("returning queries: ");
+    LOGF(getLogger(), LOG_LVL_DEBUG, "Returning chunk queries: ");
     for(unsigned int t=0;t<q.size();t++){
-        LOGF_DEBUG("%1%" % q.at(t));
+        LOGF(getLogger(), LOG_LVL_DEBUG, "%1%" % q.at(t));
```

Comment:
This whole loop and `LOGF` before it can also be combined in one `LOGF()` if you use your new `printable()`